src/fhir/sdc-import-common.js

```diff
@@ -902,7 +909,10 @@ function addCommonSDCImportFns(ns) {
     if (mismatchItem.length > 0) {
       console.log('Mismatched items linkId:', mismatchItem);
     }
-    qr.item[0].item = itemsInBoth;
+
+    // Update questionnaire response items with filtered items
+    qr.item[0].item ? qr.item[0].item = itemsInBoth : qr.item = itemsInBoth;
```

Comment:
This is confusing to have assignment expressions in a ternary operator. Can we use if-then instead?